packages/angular/src/builders/utilities/webpack.ts

```diff
@@ -22,11 +23,44 @@ export async function mergeCustomWebpackConfig(
 
   // The extra Webpack configuration file can export a synchronous or asynchronous function,
   // for instance: `module.exports = async config => { ... }`.
+  let newConfig: any;
   if (typeof config === 'function') {
-    return config(baseWebpackConfig, options, target);
+    newConfig = config(baseWebpackConfig, options, target);
   } else {
-    return merge(baseWebpackConfig, config);
+    newConfig = merge(baseWebpackConfig, config);
   }
+
+  // license-webpack-plugin will at times try to scan the monorepo's root package.json
+  // This will result in an error being thrown
+  // Ensure root package.json is excluded
+  const licensePlugin = newConfig.plugins.find(
+    (p) => p.constructor.name === 'LicenseWebpackPlugin'
+  );
+  if (licensePlugin) {
+    let rootPackageJsonName: string;
+    const pathToRootPackageJson = join(
+      newConfig.context.root ?? workspaceRoot,
+      'package.json'
+    );
+    if (existsSync(pathToRootPackageJson)) {
+      try {
+        const rootPackageJson = JSON.parse(
+          readFileSync(pathToRootPackageJson, 'utf-8')
+        );
+        rootPackageJsonName = rootPackageJson.name;
+        licensePlugin.pluginOptions.excludedPackageTest = (pkgName: string) => {
+          if (!rootPackageJsonName) {
```

Comment:
In the case where the root `package.json` does not contain `name`, then this error could still happen?

Reply:
If someone runs into this and doesn't have a name in the package.json, we can suggest they set one